CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ set(VARS_PREFIX "iDynTree")
 
 set(${VARS_PREFIX}_MAJOR_VERSION 0)
 set(${VARS_PREFIX}_MINOR_VERSION 11)
-set(${VARS_PREFIX}_PATCH_VERSION 1)
```

Comment:
I was thinking to stick to patch version 1 because I still need to release 0.11.1 .